engine/runtime/src/main/scala/org/enso/compiler/codegen/IrToTruffle.scala

```diff
@@ -326,12 +324,12 @@ class IrToTruffle(
               language,
               expressionProcessor.scope,
               moduleScope,
-              () => bodyBuilder.bodyNode(),
+              bodyBuilder.bodyNode(),
```

Comment:
The `LazyBodyNode` has been introduced by #3429 and according to [the measurements](https://www.pivotaltracker.com/n/projects/2539304/stories/182024911) it speeds the IDE startup by 21%:

> 1.5s out of 7s speedup

I don't think we can give that up. We need to find a way to convert nodes lazily and make it work even with `int` based slots. It shouldn't be that hard - traversing `IR` shall give us the exact amount of needed slots without creating the Truffle nodes.